Source/WebCore/Modules/geolocation/Geolocation.cpp

```diff
@@ -434,7 +434,7 @@ void Geolocation::makeCachedPositionCallbacks()
     for (auto& notifier : m_requestsAwaitingCachedPosition) {
         // FIXME: This seems wrong, since makeCachedPositionCallbacks() is called in a branch where
         // lastPosition() is known to be null in Geolocation::setIsAllowed().
-        notifier->runSuccessCallback(lastPosition());
+        notifier->runSuccessCallback(*lastPosition());
```

Comment:
What guarantees `lastPosition()` isn't `nullptr` here? The FIXME before this line suggests that it is likely to be nullptr.